modules/govuk_jenkins/files/var/lib/jenkins/groovy_scripts/govuk_jenkinslib.groovy

```diff
@@ -526,4 +540,25 @@ def hasDatabase() {
   sh(script: "test -e config/database.yml", returnStatus: true) == 0
 }
 
+/**
+ * Manually set build status in Github.
+ *
+ * Useful for downstream builds that want to report on the upstream PR.
+ *
+ * @param repoName Name of the repo being built
+ * @param commit SHA of the triggering commit on govuk-content-schemas
+ * @param message The message to report
+ * @param state The build state: one of PENDING, SUCCESS, FAILED
+ */
+def setBuildStatus(repoName, commit, message, state) {
```

Comment:
Yes, that's the intention - see https://github.com/alphagov/govuk-content-schemas/pull/572 for an example.